lib/mutex_m.rb

```diff
@@ -109,8 +109,17 @@ def mu_initialize # :nodoc:
     @_mutex = Thread::Mutex.new
   end
 
-  def initialize(*args) # :nodoc:
-    mu_initialize
-    super
+  if RUBY_VERSION < '2.7'
+    def initialize(*args) # :nodoc:
+      mu_initialize
+      super
+    end
+  else
+    eval <<~RUBY, binding, __FILE__, __LINE__ + 1
+      def initialize(...) # :nodoc:
+        mu_initialize
+        super
+      end
+    RUBY
   end
```

Comment:
As an alternative, we can use `ruby2_keywords` instead.

```suggestion
  end
  ruby2_keywords(:initialize) if respond_to?(:ruby2_keywords, true)
```

Reply:
I agree with @kamipo that `ruby2_keywords` is the best way to handle this.

Reply:
Thanks for your suggestion! I applied the patch in https://github.com/ruby/mutex_m/pull/4/commits/f55360b7399086e3adaa56e2d409e4e5d23c2e77